doc_src/en/First_Steps.xml

```diff
@@ -0,0 +1,91 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE book PUBLIC "-//OASIS//DTD DocBook XML V4.5//EN"
+"../../../docbook-xml-4.5/docbookx.dtd"
+[<!ENTITY % manualvariables SYSTEM "manualvariables.mod"> %manualvariables;]>
+<article lang="en" id="first.steps">
+  <title>OmegaT &vernb;</title>
+  <section>
+	<title>First steps</title>
+	<para>Hit <keycap>F1</keycap> or access the <guimenu>> Help</guimenu> menu
+	to open the user manual. It comes with a thorough introduction to OmegaT. To
+	start using OmegaT with minimal settings, do this:
+	
+	<orderedlist>
+	  <listitem>
+		<para>Use <guimenu>> Project</guimenu> <guimenuitem>>
+		New...</guimenuitem> to create a new project.</para>
+	  </listitem>
+	  <listitem>
+		<para>Change the language codes to reflect your source and target
+		languages. Click on <guibutton>OK</guibutton>.</para>
+
+		<para>→ OmegaT has created a folder set that will contain your files. It
+		also has created an empty translation memory that it will fill in the
+		background, one segment at as time as you type your translation. Matches
+		for that memory will eventually appear in the <guilabel>Fuzzy
+		Match</guilabel> pane.</para>
+	  </listitem>
+	  <listitem>
+		<para>Click on <guibutton>Add Files...</guibutton> and add the files
+		that you want to translate. Dismiss the window.</para>
+
+		<para>→ OmegaT has copied the selected files into the project that you
+		created in the previous step. It now displays them segment per segment
+		for you to translate them.</para>
+	  </listitem>
+	  <listitem>
+		<para>Type the translation of the first segment. Hit
+		<keycap>Enter</keycap>.</para>
+
+		<para>→ OmegaT has registered that segment into the project translation
+		memory and has moved the cursor to the next segment.</para>
+	  </listitem>
+
+	  <listitem>
+		<para>Use <guimenu>> Project</guimenu ><guimenuitem>> Create Translated
+		Files</guimenuitem> to create the translated document at any
+		time.</para>
+
+		<para>→ OmegaT has stored the translate files into the
+		<guilabel>target</guilabel> folder of the project that was created in
+		the first step. Use <guimenu>> Project</guimenu> <guisubmenu>> Access
+		Project Contents</guisubmenu> to access the
+		files.</para>
+	  </listitem>
+
+	  <listitem>
+		<para>If you want to know more, read the user manual.</para>
+	  </listitem>
+	</orderedlist>
+	</para>
+  </section>
+
+  <section>
+	<title>Freedoms</title>
+	<para>OmegaT is a free translation memory solution for professional
```

Comment:
Replace "translation memory" with "computer-assisted translation (CAT)".
(The abbreviation isn't strictly necessary, but I prefer to include it. That's purely preference, though.)